docs/docs/rasa-sdk-changelog.mdx

```diff
@@ -0,0 +1,5 @@
+---
+id: rasa-sdk-changelog
+sidebar_label: Rasa SDK Changelog
+title: Rasa SDK Changelog
+---
```

Comment:
do we have a plan for this?

Reply:
it exists already here: https://github.com/RasaHQ/rasa-sdk/blob/master/CHANGELOG.rst

AFAIK it would be easily integrated if we were to source the SDK docs from the SDK as planned, otherwise not so easy (hence it not being in yet). 

I was under the impression we could release these docs in a WIP stage during the alpha (hence an empty page here and there, to keep track of what needs to be done), but I could also remove it for now, either way works.

Reply:
For now, i'll add the pages, create a TODO on them, but remove them from the sidebar

Reply:
> I was under the impression we could release these docs in a WIP stage during the alpha
yeah definitely!